### PR TITLE
Border radius tokens

### PR DESCRIPTION
**Pull request by yceballost**
https://www.figma.com/file/PApssATUkePZnIOB4GmAeI/%F0%9F%94%B9-%E2%80%8E%E2%80%8ETokens-definition?node-id=1549-274&t=Q6WrsApCMZ1BTv5W-4

### CODE REVIEW

File: tokens/vivo-new.json
Comment (yceballost) [inline tokens/vivo-new.json:1]: @BrunoBiagioniNeto please, confirm this values for vivo new skin

You can check what radius applies to components here
https://www.figma.com/file/PApssATUkePZnIOB4GmAeI/%F0%9F%94%B9-%E2%80%8E%E2%80%8ETokens-definition?node-id=1549-274&t=Q6WrsApCMZ1BTv5W-4

File: tokens/blau.json [hunk 1]
```diff
@@ -953,6 +953,44 @@
...
+    "container": {
+      "value": "16",
+      "type": "borderRadius"
+    },
+    "indicator": {
+      "value": "999",
+      "type": "borderRadius"
+    },
+    "input": {
+      "value": "4",
```
Comment (AnaMontes11) [inline tokens/blau.json]: "input": {
      "value": "8",

File: tokens/movistar-classic.json [hunk 2]
```diff
@@ -1405,6 +1405,44 @@
...
+    },
+    "container": {
+      "value": "16",
+      "type": "borderRadius"
+    },
+    "indicator": {
+      "value": "999",
+      "type": "borderRadius"
+    },
+    "input": {
```
Comment (AnaMontes11) [inline tokens/movistar-classic.json:1468]: "input": {
      "value": "8",

File: tokens/movistar.json [hunk 3]
```diff
@@ -953,6 +953,44 @@
...
+    },
+    "container": {
+      "value": "16",
+      "type": "borderRadius"
+    },
+    "indicator": {
+      "value": "999",
+      "type": "borderRadius"
+    },
+    "input": {
```
Comment (AnaMontes11) [inline tokens/movistar.json:1011]: "input": {
"value": "8",

File: tokens/o2.json [hunk 4]
```diff
@@ -953,6 +953,44 @@
...
+    },
+    "container": {
+      "value": "16",
+      "type": "borderRadius"
+    },
+    "indicator": {
+      "value": "999",
+      "type": "borderRadius"
+    },
+    "input": {
```
Comment (AnaMontes11) [inline tokens/o2.json:1011]: "input": {
"value": "8",

File: tokens/solar-360.json [hunk 5]
```diff
@@ -538,6 +538,44 @@
...
+    },
+    "container": {
+      "value": "16",
+      "type": "borderRadius"
+    },
+    "indicator": {
+      "value": "999",
+      "type": "borderRadius"
+    },
+    "input": {
```
Comment (AnaMontes11) [inline tokens/solar-360.json]: "input": {
"value": "8",

**Review by AnaMontes11 — APPROVED** [review]
Looks great! 🏁 🏁

File: tokens/solar-360.json
Comment (aweell) [inline tokens/solar-360.json:1]: Should the radius in solar be after dark instead of before like other skins?

Comment (yceballost) [reply tokens/solar-360.json:1]: absolutely! my mistake

**Review by aweell — APPROVED** [review]

File: tokens/vivo.json
Comment (yceballost) [inline tokens/vivo.json:1]: The current skin keeps the same definition as today

**Comment by BrunoBiagioniNeto** [comment]
Solo el Sheet cambia a 16 16 0 0.

**Review by BrunoBiagioniNeto — APPROVED** [review]
Validado.

**Pull request merged**